lib/stripe.js

```diff
@@ -310,6 +318,12 @@ Stripe.prototype = {
     return this._enableTelemetry;
   },
 
+  buildDefaultAgent: function(protocol) {
```

Comment:
Should this be a prefixed with an underscore? I don't _think_ we want/expect users to call this (instead just `setHttpAgent`)...

Reply:
Yeah, I wasn't sure about this. Let's prefix it.